test/witservice.go

```diff
@@ -0,0 +1,29 @@
+package test
+
+import (
+	"context"
+	account "github.com/fabric8-services/fabric8-auth/account/repository"
+	"github.com/fabric8-services/fabric8-auth/app"
+	"github.com/fabric8-services/fabric8-auth/wit"
+	"github.com/goadesign/goa/uuid"
+)
+
+// DevWITService is the default dev service implementation for WIT.
```

Comment:
I think it would be better to use `go-vcr` (or another http mocking lib) so that we don't need to introduce such a `DevWITService` in our tests. Instead, we could try to configure the `Application` with a custom HTTP client that would be used for all remote communications (here, `WIT`) and the tests would only have to provide a set of HTTP interactions (if we use go-vcr, for example). WDYT?

Reply:
opened https://github.com/fabric8-services/fabric8-auth/issues/575